components/dialog/dialog-fullscreen.js

```diff
@@ -25,68 +27,138 @@ class DialogFullscreen extends LocalizeCoreElement(AsyncContainerMixin(DialogMix
 			 * Whether to render a loading-spinner and wait for state changes via AsyncContainerMixin
 			 */
 			async: { type: Boolean },
-			_hasFooterContent: { type: Boolean, attribute: false }
+			_hasFooterContent: { type: Boolean, attribute: false },
+			_icon: { type: String, attribute: false }
 		};
 	}
 
 	static get styles() {
 		return [ dialogStyles, heading3Styles, css`
 
-			.d2l-dialog-header {
-				padding-bottom: 15px;
+			.d2l-dialog-footer.d2l-footer-no-content {
+				display: none;
 			}
 
-			.d2l-dialog-header > div > d2l-button-icon {
-				flex: none;
-				margin: -4px -15px 0 15px;
+			.d2l-dialog-content-loading {
+				text-align: center;
 			}
 
-			:host([dir="rtl"]) .d2l-dialog-header > div > d2l-button-icon {
-				margin-left: -15px;
-				margin-right: 15px;
-			}
+			@media (min-width: 616px) {
+				
+				.d2l-dialog-header {
+					border-bottom: 1px solid var(--d2l-color-galena);
```

Comment:
Just took this for a spin... the borders (header and footer) looks dark.  I think this is supposed to be gypsum.

Reply:
Yeah I also thought these looked dark, but the design _says_ galena which may be a mistake... @geurts?

Reply:
Yep, it should be gypsum.

Reply:
Ah, and I see the design has already be updated now... nevermind Jeff! 😄